From a560bcbea943cb317e96c15c608c5e2f66e5698f Mon Sep 17 00:00:00 2001
From: Stephan Bergmann <sbergman@redhat.com>
Date: Thu, 31 May 2018 14:14:48 +0200
Subject: --enable-gio also for --disable-gtk --enable-gtk3
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

...as is used by the Flatpak build.  Other settings only checked in that block
controlled by ENABLE_GTK, and thus potentially erronously left empty for
--disable-gtk --enable-gtk3 builds, are GTHREAD- and GTK_PRINT-related settings.
GTHREAD_LIBS is explicitly used in vcl/Library_vclplug_gtk3.mk even, but it
being empty apparently has no negative effects, so leave it at that for now.
On the GTK_PRINT-related settings, Caolan commented on IRC:  "ENABLE_GTK_PRINT
is probably best removed entirely I guess, it didn't work out and they're
redoing the dialog in upstream gtk again I'm told."

Change-Id: I7e026c0ac9b23e7ace2c2e92390bdcc3be7d39a3
Reviewed-on: https://gerrit.libreoffice.org/55127
Tested-by: Jenkins <ci@libreoffice.org>
Reviewed-by: Stephan Bergmann <sbergman@redhat.com>
(cherry picked from commit b5dd045bf533a2ba4d1c98debe5a2acba4c8b486)
Reviewed-on: https://gerrit.libreoffice.org/55164
Reviewed-by: Caolán McNamara <caolanm@redhat.com>
Tested-by: Caolán McNamara <caolanm@redhat.com>
---
 configure.ac | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index e35db48..9da61d9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -9875,7 +9875,9 @@ if test  "$test_gtk" = "yes"; then
         GTK_PRINT_CFLAGS=$(printf '%s' "$GTK_PRINT_CFLAGS" | sed -e "s/-I/${ISYSTEM?}/g")
         FilterLibs "${GTK_PRINT_LIBS}"
         GTK_PRINT_LIBS="${filteredlibs}"
+    fi
 
+    if test "$ENABLE_GTK" = "TRUE" || test "$ENABLE_GTK3" = "TRUE"; then
         AC_MSG_CHECKING([whether to enable GIO support])
         if test "$_os" != "WINNT" -a "$_os" != "Darwin" -a "$enable_gio" = "yes"; then
             dnl Need at least 2.26 for the dbus support.
-- 
cgit v1.1

